extensions/resource-deployment/src/common/optionSources.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as rd from 'resource-deployment';
+import * as loc from '../localizedConstants';
+
+export class OptionsSources {
```

Comment:
Not a big fan of static classes like this. We already have a pattern of declaring "services" that handle this kind of stuff - I'd rather we keep consistent and keep doing that.

Reply:
Ok will change.